### PR TITLE
Creates blockquote 

### PR DESCRIPTION
**Pull request by varshika15**

### CODE REVIEW

File: client/app/bundles/shared/components/Blockquote.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.App {
+  text-align: center;
+  background-color: rgba(0, 0, 0, 0.25);
+  font-family: Hiragino Sans, Lato ;
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.css]: Let's make Lato come first here and not use Hiragino Sans! Could probably be `Lato, sans-serif`.

File: client/app/bundles/shared/components/Blockquote.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export default class Blockquote extends React.Component {
+  render() {
+    return (
+      <div className="Blockquote">      
+        
+        <blockquote>"It's not just all in your head, it's all around you. We 
+          can heal together."</blockquote>
+          <blockquote>{"\u2764\uFE0F"}</blockquote>
+        
+      </div>
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.jsx]: A few things here!

1. Let's make sure this is just one blockquote element and not multiple.
2. Let's pass in the text as props.
3. Let's remove quotes from the JSX here and instead have the CSS generate them using pseudo elements!

**Comment by varshika15** [comment]
Sure, I will make the these changes.

**Comment by julianguyen** [comment]
Hey, any updates on this? :)

**Comment by prateksha** [comment]
Hey, @varshika15 
If you aren't working on this issue, can I take it up?

**Pull request closed without merging**